scripts/index-api/index-api.js

```diff
@@ -329,45 +286,108 @@ async function scrapeAndIndex() {
   // Summary
   const summary = {
     total: allRecords.length,
-    groups: allRecords.filter(r => r.type === 'API Group').length,
-    endpoints: allRecords.filter(r => r.type === 'API Endpoint').length,
-    withMethod: allRecords.filter(r => r.method && r.method !== 'UNKNOWN').length,
-    withPath: allRecords.filter(r => r.path && r.path !== 'UNKNOWN').length,
-    selfManaged: allRecords.filter(r => r.product === 'Self-Managed').length,
-    cloud: allRecords.filter(r => r.product === 'Cloud').length
+    apis: apiRootRecords.length,
+    groups: allRecords.filter((r) => r.type === 'API Group').length,
+    endpoints: allRecords.filter((r) => r.type === 'API Endpoint').length,
+    withMethod: allRecords.filter((r) => r.type === 'API Endpoint' && r.method && r.method !== 'UNKNOWN').length,
+    withPath: allRecords.filter((r) => r.type === 'API Endpoint' && r.path && r.path !== 'UNKNOWN').length,
+    selfManaged: allRecords.filter((r) => r.product === 'Self-Managed').length,
+    cloud: allRecords.filter((r) => r.product === 'Cloud').length,
```

Comment:
Consider adding the types as consts. And force to-lower case to defend yourself. Also, due to the big number of tests, consider using reduce. Something like that... adjust to match your needs 

```suggestion
const API_GROUP = 'API Group';
const API_ENDPOINT = 'API Endpoint';
const SELF_MANAGED = 'Self-Managed';
const CLOUD = 'Cloud';
const UNKNOWN = 'UNKNOWN';

const stats = allRecords.reduce((acc, record) => {
  const type = record.type?.toLowerCase();
  const product = record.product?.toLowerCase();
  const method = record.method?.toUpperCase();
  const path = record.path?.toUpperCase();

  if (type === API_GROUP.toLowerCase()) acc.groups++;
  if (type === API_ENDPOINT.toLowerCase()) {
    acc.endpoints++;
    if (method && method !== UNKNOWN) acc.withMethod++;
    if (path && path !== UNKNOWN) acc.withPath++;
  }
  if (product === SELF_MANAGED.toLowerCase()) acc.selfManaged++;
  if (product === CLOUD.toLowerCase()) acc.cloud++;
  return acc;
}, {
  groups: 0,
  endpoints: 0,
  withMethod: 0,
  withPath: 0,
  selfManaged: 0,
  cloud: 0
});
```